src/gather_stats.ts

```diff
@@ -106,11 +120,8 @@ async function retrieve_nodes_from_network() {
             headers: { authorization: `token ${process.env.SECRET_GITHUB_PERSONAL_ACCESS_TOKEN}` },
         });
 
-        const rate_limit = {
-            spent: null as null | number,
-            remaining: null as null | number,
-            limit: null as null | number,

```

Comment:
`null as null` is funny.

Reply:
Yeah, it looks a little strange :joy_cat: The type is `null | number`, so this should be read with `as` having lower/weaker precedence. This pattern still occurs in `null as null | cliProgress.SingleBar` - the order is intended to indicate "this is `null` now, but will be a specific type later".

Elsewhere, I wrote `closedAt: string | null`. There, the order indicates "this is usually `string` but can sometimes be `null`".